src/encoded/static/components/software.js

```diff
@@ -125,20 +123,6 @@ var Software = module.exports.Software = React.createClass({
 });
 globals.content_views.register(Software, 'Software');
 
-// Commenting out until pipelines are used.
-
-var PipelinesUsingSoftwareVersion = module.exports.PipelinesUsingSoftwareVersion = React.createClass({
-    render: function () {
-        var context = this.props.context;
-        return (
-            <div>
-                <h3>Pipelines using software {context.title}</h3>
-                <PipelineTable {...this.props} softwareId={context['@id']}/>
-            </div>
-        );
-    }
-});
-
```

Comment:
Note that you can resolve a circular dependency by moving the `require(...)` from the module level to the function in which you use it.